lms/templates/courseware/info.html

```diff
@@ -89,15 +88,47 @@ <h2 class="page-subtitle">${course.display_name_with_default}</h2>
           % endif
         </section>
         <section aria-label="${_('Handout Navigation')}" class="handouts">
-          % if SelfPacedConfiguration.current().enable_course_home_improvements:
+
+      % if SelfPacedConfiguration.current().enable_course_home_improvements:
             <h1 class="handouts-header">${_("Important Course Dates")}</h1>
-            ${HTML(get_course_date_summary(course, user))}
-          % endif
+            ## Should be organized by date, last date appearing at the bottom
 
-          <h1 class="handouts-header">${_(course.info_sidebar_name)}</h1>
-          ${HTML(get_course_info_section(request, masquerade_user, course, 'handouts'))}
+            <%
+                sorted_date_blocks = get_course_dateblocks(course, user)
```

Comment:
Is this temporary variable needed?  The code would read better here if `get_course_date_blocks` was just called inline in the `for` loop below.  (It's not a performance issue if that's what you were worried about.  Since the function would still be called only once.)